### PR TITLE
use reusable bit setup action

### PR DESCRIPTION
**Pull request by ocombe**
I created a [reusable Github action](https://github.com/teambit/setup-action) that does all of the setup for bit, I tested it on the bit-angular repo and it works fine.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,22 +10,11 @@ jobs:
...
-          bit config set interactive false
-          bit config set analytics_reporting false
-          bit config set anonymous_reporting false
-          bit config set user.token ${{ env.BIT_TOKEN }}
-          bit config set no_warnings true
-          bit config set error_reporting true
-          bit config set user.name symphony_backend
+      - uses: teambit/setup-action@v1
+        with:
+          name: symphony_backend
```
Comment (KutnerUri) [inline .github/workflows/ci.yaml:15]: @ocombe - maybe name it `design_scope`?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -10,22 +10,11 @@ jobs:
...
-      - name: Bit config
-        run: |
-          bit config set interactive false
-          bit config set analytics_reporting false
-          bit config set anonymous_reporting false
-          bit config set user.token ${{ env.BIT_TOKEN }}
-          bit config set no_warnings true
-          bit config set error_reporting true
-          bit config set user.name symphony_backend
+      - uses: teambit/setup-action@v1
```
Comment (KutnerUri) [inline .github/workflows/ci.yaml:13]: are you sure this can come before actions/checkout?

**Review by KutnerUri — APPROVED** [review]

**Comment by KutnerUri** [comment]
I'm going to work on this repo today, I will fix the lint

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (ocombe) [reply .github/workflows/ci.yaml:13]: it seems to work fine on the 2 other repositories that use it 🙂 what do you think could be the issue there?

**Pull request merged**